### DISCUSSION TITLE
IP addresses Status meaning

### DISCUSSION THREAD

**Discussion by Sispheor** [netbox-community/netbox#6514] [Q&A] [answered]
Hi, 

I have a question. In the doc there is no description about IP adresses status.
For example I'm a bit confusing with the difference between DHCP and available (when the IP is not referenced in Netbox)

Is this a correct description?

**Active**: Attached to an interface and currently in use
**Reserved:** Not yet attached to an interface but marked as reserved so we don't use it
**Deprecated:** In use but once free it should not be used anymore. Or not an address to use like a broadcast address
**DHCP:** Available for the DHCP. Like when the ip is not declared and so "available" ? Once taken by somebody it should switch to active?

Many thanks for your time.

**Comment by candlerb** [comment, marked as answer]
"DHCP" = "Part of a DHCP pool".  That is, you can define a DHCP pool by creating a bunch of adjacent IP addresses with status "DHCP".

It would *not* switch to Active when in use; it's still part of the DHCP pool (i.e. for use by clients which are not otherwise documented in Netbox).  If you want to see which IPs in the pool are in use, you should query your DHCP server.

A proposal for a better way of defining DHCP pools is in #834

**Reply by Sispheor** [reply]
Ok so DHCP is kind of reserved. Reserved by a software.
